packages/utils/src/route.js

```diff
@@ -173,6 +174,18 @@ export const createRoutes = function createRoutes ({
         }
       }
     })
+    if (typeof trailingSlashes !== 'undefined') {
```

Comment:
Previous comment was swollen by outdated comment
```suggestion
if (trailingSlashes !== undefined) {
  route.pathToRegexpOptions = { ...route.pathToRegexpOptions, strict: true }
  route.path = route.path.replace(/\/+$/, '') + (trailingSlashes ? '/' : '')
}
```